### ISSUE TITLE
Full Height Hero content full width with container

### ISSUE THREAD

**Issue by gRoberts84** [jgthms/bulma#688]
### Overview of the problem

This is about the Bulma **CSS framework**

I'm using Bulma **version** [0.4.0]
My **browser** is: Chrome [57.0.2987.133 (64-bit)], Firefox [53.0 (64-bit)], Safari [10.0.3 (12602.4.8)]
This is a **Sass** issue: I'm using version [3.4.22]
I am sure this issue is **not a duplicate**?

### Description

When using a full height hero, the container max-width is ignored by the flexbox configuration.

Using the included CSS file within the node_modules/bulma/css/bulma.css also has the same issue, however using that of within the repository works fine.

### Steps to Reproduce

1. Install Bulma NPM package (0.4.0)
2. Create Sass file linking to node_modules/bulma/sass/bulma.scss
3. Compile Sass file.
4. Create html document with full height hero and link to compiled CSS file.

```<!doctype html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <meta http-equiv="X-UA-Compatible" content="ie=edge">
        <title>Document</title>
        <link rel="stylesheet" href="assets/css/site.min.css">
    </head>
    <body>

        <section class="hero is-success is-fullheight">
            <div class="hero-body">
                <div class="container">
                    // Text content goes here (h1 and call to action etc...)
                </div>
            </div>
            <div class="hero-foot">
                <div class="has-text-centered">
                    // Scroll down arrow here?
                </div>
            </div>
        </section>

        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum libero maiores perferendis quas? Ab at autem
            eius exercitationem expedita ipsa laudantium maxime sit! Architecto atque nam neque nobis odit porro.</p>

        <p>Ab atque aut consequatur culpa dignissimos doloribus eligendi eveniet ipsa maiores nihil non obcaecati odio
            odit officiis optio, perferendis perspiciatis placeat provident quas quibusdam repellat sequi soluta sunt
            voluptates, voluptatibus!</p>

        <p>Adipisci aliquid asperiores cupiditate dignissimos distinctio dolorem, doloremque dolores dolorum eligendi ex
            explicabo fuga impedit in ipsam itaque maxime minima omnis perspiciatis porro quas, qui quia quos tenetur
            veniam voluptates.</p>

        <p>Aspernatur deleniti facilis iure magnam quasi quidem quo rem voluptatem voluptatibus. Aspernatur consequuntur
            dolore, est exercitationem explicabo facere impedit ipsa labore magni nostrum quas repudiandae tempora
            tempore ullam voluptatibus. Dicta.</p>

        <p>Animi aspernatur blanditiis, deleniti dicta dolorum eius eligendi est eveniet ipsa laudantium nam officiis
            porro quam sapiente, sunt ullam vitae? Adipisci consequatur cumque earum illo maiores nemo odit porro
            quaerat.</p>

    </body>
</html>
```

### Expected behavior

Container max-width should be honoured and display accordingly.

### Actual behavior

Container is the full width of the view-port unless you disable the following CSS:

    .hero.is-fullheight .hero-body>.container {
        flex-grow: 1;
        flex-shrink: 1;
    }

![screencapture-localhost-8080-1492673622320](https://cloud.githubusercontent.com/assets/348184/25218837/37bc9f82-25a4-11e7-92aa-c74334e37904.png)

**Comment by jgthms** [comment]
Dupe. And fixed in `0.4.1`.

**Comment by gRoberts84** [comment]
Hey @jgthms 

Any chance you could update the NPM package as this is still at 0.4.0?

Thanks
